recipes/metamorpheus/build.sh

```diff
@@ -3,13 +3,13 @@ set -eox pipefail
 
 PREFIX=$(echo "${PREFIX}" | tr '\\' '/')
 
-mkdir -p $PREFIX/bin
-cp -r $SRC_DIR/* $PREFIX
+mkdir -p $PREFIX/bin/tools
+cp -r $SRC_DIR/* $PREFIX/bin
```

Comment:
This just copies all your application files into the `bin` directory which doesn't seem like a great idea - it's not well encapsulated and will inevitably lead to files being clobbered. Instead, I think your application should be installed into it's own folder (somewhere) rather than into a common folder. I'd suggest `$DOTNET_ROOT/tools/metamorpheus`

Reply:
Thanks for all the advice. I'm still learning how these end up in relation to other packages.

This makes sense with your suggested change below regarding where the DLL should be. I'll go ahead and make those changes.